tools/cbmc/proofs/ARP/ARP_FreeRTOS_OutputARPRequest/config_CBMC_PROOF_ASSUMPTION_DOES_NOT_HOLD/Makefile.json

```diff
@@ -1,15 +1,14 @@
 {
```

Comment:
This file gets generated from Configurations.json, right? If so, please remove it from this PR